### PR TITLE
feat: intro-to-anchor-draft

### PR DESCRIPTION
**Pull request by ZYJLiu**

### CODE REVIEW

File: content/intro-to-anchor.md [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+
+- Use the Anchor framework to build a basic program
+- Describe the basic structure of an Anchor program
+- Explain how to implement basic account validation and security checks with Anchor
+
+# TL;DR
+
+- **Anchor** is a framework building Solana programs
+- **Anchor** macros speed up the process of building Solana programs by abstracting away a significant amount of boilerplate code
+- **Anchor** allows you more easily build secure programs by performing certain security checks, requiring account validation, and providing a simply way to implement additional checks.
```
Comment (ixmorrow) [inline content/intro-to-anchor.md:15]: From -> **Anchor** allows you more easily
To: **Anchor** allows you to more easily

Comment (ixmorrow) [inline content/intro-to-anchor.md:15]: Typo:
From -> and providing a simply way to implement additional checks.
To -> and providing a simple way to implement additional checks.

File: content/intro-to-anchor.md [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+
+# Overview
+
+## What is Anchor?
+
+Anchor is a framework building Solana programs. The Anchor framework organizes a program in distinct sections that separates the instruction logic from account validation and security check. This separation generally allows for easier readability.
```
Comment (ixmorrow) [inline content/intro-to-anchor.md:21]: Anchor is a framework for building Solana programs. The Anchor framework organizes a program into distinct sections that separates the instruction logic from account validation and security checks.

File: content/intro-to-anchor.md [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+
+Anchor also enables you to quickly build Solana programs by abstracting away various tasks such as serialization and deserialization of account. This is accomplished by bundling various boilerplate code into macros, allowing you to focus on the business logic of your program. Additionally, Anchor is designed to handling many common security checks while allowing you to easily define additional checks to help you build more secure programs.
```
Comment (ixmorrow) [inline content/intro-to-anchor.md:23]: Anchor also enables you to quickly build Solana programs by abstracting away various tasks such as the serialization and deserialization of an account.  This is accomplished by bundling boilerplate code into macros, allowing you to focus on the business logic of your program. Additionally, Anchor is designed to inherently handle many common security checks while allowing you to easily define additional checks to help you build more secure programs.

The organization and abstraction capabilities of Anchor, make Solana programs much easier to read and write.

File: content/intro-to-anchor.md [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+pub struct AccountStruct {
+		data: u64
+}
+```
+
+You are now ready to build your own Solana program using the Anchor framework!
+
+# Demo
+
+We're going to build a simple Anchor program using Solana Playground. The Solana Playground is a tool that allows you to write and deploy Solana programs from the browser.
```
[the remark this answers is not in the document]
Comment (ZYJLiu) [reply content/intro-to-anchor.md:318]: Yea for sure. Using SolPg just for this first intro lesson.
Local setup next lesson. 
Following local setup lesson will be rebuilding movie review program in Anchor locally.

File: content/intro-to-anchor.md [hunk 5]
```diff
@@ -0,0 +1,511 @@
...
+Now it’s your turn to build something independently. Because we're starting with very simple programs, yours will look almost identical to what we just created. It's useful to try and get to the point where you can write it from scratch without referencing prior code, so try not to copy and paste here.
+
+1. Write a new program that initializes a `counter` account and set the `count` field using the an instruction data argument passed into the instruction
+2. Implement both an `increment` and `decrement` instruction
+3. Build and deploy your program like we did in the demo
+4. Test your newly deployed program and use Solana Explorer to check the program logs
+
+As always, get creative with these challenges and take them beyond the basic instructions if you want - and have fun!
+
+Try to do this independently if you can! But if you get stuck, feel free to reference the [solution code](https://beta.solpg.io/62f14c6df6273245aca4f636).
```
[the remark this answers is not in the document]
Comment (ZYJLiu) [reply content/intro-to-anchor.md:511]: Previous solution code is for demo.
Last line solution code is for challenge. 

The initialize instruction is slightly different between the two.
Initialize instruction in challenges requires an `input` parameter as instruction data

**Review by jamesrp13 — CHANGES_REQUESTED** [review]
@ZYJLiu have a look at my comments, but also look at my commit on your branch and tell me what you think. If you're good with it lmk and I'll approve and merge

**Pull request merged**